.travis.yml

```diff
@@ -33,9 +33,8 @@ jobs:
     - stage: Linters
```

Comment:
This commit looks like it belongs to one of the previous ones

Reply:
Can you please tell me which commits you want to get squashed?

Reply:
This one: https://github.com/openSUSE/open-build-service/pull/4853/commits/e54545894df03bb1cc93d60a35cbc6e0ae5e6629